### ISSUE TITLE
Inheriting same function from multiple interfaces requires override

### ISSUE THREAD

**Issue by hfhbd** [rickclephas/KMP-NativeCoroutines#51]
Hey,
using an interface with the same functions from multiple interfaces works with plain Kotlin, but not with your plugin.
```kt
import kotlinx.coroutines.flow.*

interface A {
    fun foo(): Flow<Int>
}

interface B {
    fun foo(): Flow<Int>
}

interface C: A, B
```
Without your plugin: successful build.
With your plugin: 
`e: /Users/philipwedemann/Downloads/multiInterface/src/iosArm64Main/kotlin/Main.kt: (11, 1): Class 'C' must override public open fun fooNative(): NativeFlow<Int> /* = (onItem: (Int, Unit) -> Unit, onComplete: NativeCallback<NSError?> /* = (NSError?, Unit) -> Unit */) -> NativeCancellable /* = () -> Unit */ */ defined in A because it inherits multiple interface methods of it`

Versions:
Kotlin 1.6.10
Coroutines: 1.6.0-native-mt
com.rickclephas.kmp.nativecoroutines: 0.11.3

[multiInterface.zip](https://github.com/rickclephas/KMP-NativeCoroutines/files/8253391/multiInterface.zip)

**Comment by rickclephas** [comment]
Fix has been published in [v0.11.4](https://github.com/rickclephas/KMP-NativeCoroutines/releases/tag/v0.11.4)

**Comment by rickclephas** [comment]
FYI for the plugin to generate the native function in interface `C` you must override `foo`: https://github.com/rickclephas/KMP-NativeCoroutines/blob/c9222ddee23054680d9744d0020a619cbb142674/sample/shared/src/commonMain/kotlin/com/rickclephas/kmp/nativecoroutines/sample/issues/GH51.kt#L19

There is no need to provide an implemention, just declaring the override is enough. 
This is also true for abstract classes, though for those you will need to mark the override as `abstract`.
